### PR TITLE
Added Hill Cipher

### PR DESCRIPTION
**Pull request by debck**

### CODE REVIEW

File: hill-cipher.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    for r in result:
+                numeric_letter = r[0] % 26
+                val = chr(numeric_letter + 97)
+                unCiphered = unCiphered + val
+
+    return unCiphered
+
+
+def main():
+    code = raw_input("Enter ciphertext: ")
```
Comment (palash25) [inline hill-cipher.py:32]: `raw_input()` is no longer supported by python3 change that to `input()`

File: hill-cipher.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    print
```
Comment (palash25) [inline hill-cipher.py:33]: Why an empty `print` statement?

File: hill-cipher.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    plaintext = ""
+
+    while(code):
+        ciphertext = code[:3]
+        code = code[3:]
+        plaintext = plaintext + hill(ciphertext)
+        print plaintext
+        print
+    main()
```
Comment (palash25) [inline hill-cipher.py:42]: use dunder methods to execute `main`

**Review by palash25 — COMMENTED** [review]
I also want you post a screenshot of the output on the terminal in your PR description to verify whether its working

File: hill-cipher.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+def main():
+    code = raw_input("Enter ciphertext: ")
+    print
+    plaintext = ""
+
+    while(code):
+        ciphertext = code[:3]
+        code = code[3:]
+        plaintext = plaintext + hill(ciphertext)
+        print plaintext
```
Comment (sibasish14) [inline hill-cipher.py:40]: `print` is a function in python3. So, write like this - `print(plaintext)`

**Comment by debck** [comment]
@palash25  kK I think there r plenty of mistakes .
Actually I am new to python so finding it little different...
So shall I close this PR ..

**Comment by palash25** [comment]
@debck no never close a PR unless told so make the request changes and amend your commits using `git commit -a --amend` command and push the changes to this same PR

**Comment by debck** [comment]
@palash25  ok bhaiya ....

**Pull request closed without merging**